src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -14,7 +14,7 @@
 import seedu.address.logic.commands.DeleteCommand;
 import seedu.address.logic.commands.EditCommand;
 import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.SearchCommand;
 import seedu.address.logic.commands.HelpCommand;
 import seedu.address.logic.commands.ListCommand;
```

Comment:
Error: eckstyle] [ERROR] /Users/runner/work/tp/tp/src/main/java/seedu/address/logic/parser/AddressBookParser.java:18:1: Wrong lexicographical order for 'seedu.address.logic.commands.HelpCommand' import. Should be before 'seedu.address.logic.commands.SearchCommand'. [CustomImportOrder]

Error: eckstyle] [ERROR] /Users/runner/work/tp/tp/src/main/java/seedu/address/logic/parser/AddressBookParser.java:19:1: Wrong lexicographical order for 'seedu.address.logic.commands.ListCommand' import. Should be before 'seedu.address.logic.commands.SearchCommand'. [CustomImportOrder]